users/templates/users/index.html

```diff
@@ -1,10 +1,8 @@
 {% extends "base.html" %} {% block content %}
-
```

Comment:
Maybe you can get rid of these changes.

Reply:
I didn't find any blank line on my local IDE
![image](https://user-images.githubusercontent.com/55912169/229331744-f6d03919-9dd8-444c-8320-28e7d78a80cd.png)
That's weird

Reply:
Get rid of the change means I said you need not remove the line. You can maybe revert the change or add back the new line and add the change.